spec/features/single_content_item_spec.rb

```diff
@@ -267,13 +267,13 @@
       it 'renders the collections application' do
         stub_metrics_page(base_path: 'collections/path', time_period: :past_30_days, publishing_app: 'collections-publisher')
         visit '/metrics/collections/path'
-        expect(page).to have_link("Edit in Collections publisher", href: 'http://support.dev.gov.uk/general_request/new')
+        expect(page).to have_link("Request a content change in the GOV.UK support form", href: 'http://support.dev.gov.uk/content_change_request/new')
```

Comment:
We should reference the locales in the test too - so content designers can make changes without breaking tests.